proto/viam/robot/v1/robot.proto

```diff
@@ -389,11 +389,13 @@ message ResourceStatus {
     STATE_UNCONFIGURED = 1;
     // a resource that is being configured.
     STATE_CONFIGURING = 2;
-    // a resource that has been successfully configured once and is not being
+    // a resource that has been successfully configured once, is healthy, and is not being
```

Comment:
does healthy mean ready state? maybe we can explicitly say ready for clarity

Reply:
Yeah this is not as clear as it could be - I changed this to "not unhealthy" since we have a definition for that state. I don't want to conflate "ready" with "healthy" too much since a resource can be "healthy" (no errors) but re-configuring or removing.